src/routers/PlacesAutocompleteRouter.js

```diff
@@ -28,20 +29,33 @@ class PlacesAutocompleteRouter extends AppDevRouter<string> {
         }
 
         // not in cache
-        const response = await axios.get(
-            'https://maps.googleapis.com/maps/api/place/autocomplete/json',
-            {
-                params: {
+        const options = {
+            method: 'GET',
+            url: 'https://maps.googleapis.com/maps/api/place/autocomplete/json',
+            qs:
+                {
                     location: '42.4440,-76.5019',
                     key: process.env.PLACES_KEY,
                     input: query,
                     radius: 24140,
                     strictbounds: '',
                 },
-            },
-        );
+        };
 
-        const { predictions } = response.data;
+        const AutocompleteRequest = JSON.parse(await new Promise((resolve, reject) => {
```

Comment:
should we write `AutocompleteRequest` in lowerCamelCase for consistency?

Reply:
👍

Reply:
+1 to alicia's comment